### PR TITLE
PR : Awesome book es6 Refactor

### PR DESCRIPTION
**Pull request by ichala**
in this PR I added the following :

- Create a new directory and initialize it with git.
- Used npm init -y command to create package.json file.
- Create the entry point for your JavaScript code called index.js
- Create a directory called modules.
- Re-create my Awesome books app using es6 syntax
- Keep all functionalities of the app without errors.
- Refactor my methods and functions to arrow functions.
- I made sure that I used let and const in the correct way.
- download Luxon and use it as class to show date and time

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,64 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-Thank you to Microverse Inc.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (mikethreels) [inline README.md:64]: - [x] the link to the License is not working as expected, please have a look at [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse for an example of how it is supposed to work.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-undef */
```
Comment (mikethreels) [inline index.js]: - [x] please avoid turning off the linters in your project, linters are used to optimize your code and make it more readable, therefore we advise you to keep them turned on.

File: modules/Book_Management.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+/* eslint-disable  no-restricted-globals */
```
Comment (mikethreels) [inline modules/Book_Management.js:1]: - [x] please avoid turning off the linters in your project, linters are used to optimize your code and make it more readable, therefore we advise you to keep them turned on.

File: modules/Ui.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
+/* eslint-disable  no-restricted-globals */
+/* eslint-disable  no-unused-vars */
+/* eslint-disable  no-undef */
+import ManageBooks from './Book_Management.js';
```
Comment (mikethreels) [inline modules/Ui.js]: - [x] please avoid turning off the linters in your project, linters are used to optimize your code and make it more readable, therefore we advise you to keep them turned on.

File: modules/Ui.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    const RemoveBtns = document.querySelectorAll('button');
+    const navContactBtn = document.querySelector('#nav-contact');
+    const AddBtn = document.querySelector('#add_Button');
+    const listSection = document.querySelector('#list');
+    const addNewSection = document.querySelector('#add-new');
+    const contactSection = document.querySelector('#contact');
+    const lastMod = document.querySelector('#last-modified');
+    setInterval(() => {
+      const date = luxon.DateTime.now();
+      lastMod.innerHTML = date.toLocaleString(luxon.DateTime.DATETIME_MED_WITH_SECONDS);
```
Comment (mikethreels) [inline modules/Ui.js]: - [x] Luxon should  be imported as a module as well which will look something like below:
```
import { DateTime } from "luxon";
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Chala, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Ui.js (hunk 5 above)
Comment (ichala) [reply modules/Ui.js]: @mikethreels i downloaded luxon as module ! but now I have to deal with the linters error of the js file that I downloaded ! so I will add disable linters for that  [file](https://moment.github.io/luxon/es6/luxon.js)

Comment (mikethreels) [reply modules/Ui.js]: Yes, for that file it is ok to disable the linters

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @ichala, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**